styles/base.css

```
@@ -46,33 +46,35 @@
 
   /* spectrum colors */
   --spectrum-white: rgb(255 255 255);
+  --spectrum-white--unformatted: 255, 255, 255;
```

```suggestion
  --spectrum-white--unformatted: 255 255 255;
```
I'd suggest the non-legacy (without commas) syntax for these two variables, which we're using everywhere else.